### PR TITLE
Resolve compare issues

### LINKED ISSUE #67: Compare tool removes common part of filename
This behavior may be intended (if so, ignore...), but I was pretty confused by it when trying out the tools, so I wanted to bring it up anyway.

I ran the following command to compare two of the example documents:
```
$ java -jar tools-java-1.1.0-jar-with-dependencies.jar CompareDocs compareOutput.xlsx ../testResources/SPDXJSONExample-v2.2.spdx.json ../testResources/SPDXJSONExample-v2.3.spdx.json
```
In the resulting spreadsheet, the two names were shown as `2.spdx.json` and `3.spdx.json`; the common part in the beginning was most likely trimmed by the [normalizeDocNames](https://github.com/spdx/tools-java/blob/6eea344a264c908146d6105c8647637f5e27bb08/src/main/java/org/spdx/tools/CompareSpdxDocs.java#L165) function.

### LINKED ISSUE #68: Compare tool shows diff when fields are identical
I executed
```
$ java -jar tools-java-1.1.0-jar-with-dependencies.jar CompareDocs compareOutput.xlsx ../testResources/SPDXJSONExample-v2.2.spdx.json ../testResources/SPDXJSONExample-v2.3.spdx.json
```

The resulting spreadsheet shows quite a few diffs even though the corresponding properties are identical. Not sure what the most efficient way of showing that is, so I'll attach two screenshots

Screenshot from 2022-09-09 16-40-19
Screenshot from 2022-09-09 16-41-06

### LINKED ISSUE #69: Compare tool lists package in multiple files if it is present in only one
As part of some experiments, I compared two JSON files:
```
$ java -jar tools-java-1.1.0-jar-with-dependencies.jar CompareDocs compareOutput.xlsx ../testResources/SPDXJSONExample-v2.2.spdx.json ../testResources/SPDXJSONExample-v2.3.spdx.json
```
In order to understand the output better, I tried to remove a package from either of the files and check what changed in the output. I removed this section from either of the two json files:
```
{
    "SPDXID" : "SPDXRef-fromDoap-1",
    "copyrightText" : "NOASSERTION",
    "downloadLocation" : "NOASSERTION",
    "filesAnalyzed" : false,
    "homepage" : "http://commons.apache.org/proper/commons-lang/",
    "licenseConcluded" : "NOASSERTION",
    "licenseDeclared" : "NOASSERTION",
    "name" : "Apache Commons Lang"
  }, 
```
The observed result was:
- The output was unchanged if the package was removed from only one of the two input files. The package `SPDXRef-fromDoap-1` was still shown in both columns
- When removed from both input files, the package vanished from both columns (as expected)

I would have expected no output in the v2.2 column concerning package `SPDXRef-fromDoap-1` after removing it from the v2.2 file.

### PR DESCRIPTION
**Pull request by goneall**
Resolves #67 
Resolves #68 
Resolves #69 

NOTE: This requires an update to the Spdx-Java-Library which has not yet been published.

### CODE REVIEW

**Review by nicoweidner — CHANGES_REQUESTED** [review]
Thanks for addressing these so quickly!
I found one minor issue in an edge case for the common prefix, I believe.
And the tests in `CompareSpdxDocsTest` fail now because of duplicate namespaces ;-)

File: temp.txt [hunk 1]
```diff
@@ -0,0 +1,2539 @@
+[INFO] Scanning for projects...
```
[the remark this answers is not in the document]
Comment (goneall) [reply temp.txt]: Thanks @nicoweidner for the catch - file is now removed
